compiler/noirc_evaluator/src/ssa/mod.rs

```diff
@@ -252,6 +253,7 @@ pub fn minimal_passes() -> Vec<SsaPass<'static>> {
     vec![
         // Signed integer operations need to be expanded in order to have the appropriate overflow checks applied.
         SsaPass::new(Ssa::expand_signed_checks, "expand signed checks"),
+        SsaPass::new(Ssa::expand_signed_math, "expand signed math"),
```

Comment:
```suggestion
```

Reply:
Per the doc comments: "Due to it being minimal, it can only be executed with the Brillig VM;"